core/object/class_db.h

```diff
@@ -116,12 +116,12 @@ class ClassDB {
 		HashMap<StringName, MethodInfo> signal_map;
 		List<PropertyInfo> property_list;
 		HashMap<StringName, PropertyInfo> property_map;
+		List<MethodInfo> virtual_methods;
```

Comment:
I'm not 100% certain that moving `virtual_methods` out of `DEBUG_METHODS_ENABLED` is absolutely needed to fix the linked issue, but this makes the handling of virtual methods consistent, e.g. you can find the same things through `get_method_info()`, and `get_method_list()`.